transforms/universal/fdedup/python/pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[project]
+name = "dpk_fdedup_transform_python"
+version = "0.2.2.dev1"
```

Comment:
dev2?

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/2f61be7938d7540a0a1831e85b8a961bef24d35c